core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -418,15 +418,10 @@ class TestCaseWriter {
             return "nullValue()"
         } else {
             when (resContentsItem::class) {
-                //Double::class -> return "anyOf(equalTo(${(Math.floor(resContentsItem as Double).toInt())}), closeTo(${(resContentsItem as Double)}, 0.1))"
                 Double::class -> return "numberMatches(${resContentsItem as Double})"
                 String::class -> return "containsString(\"${applyEscapes(resContentsItem as String)}\")"
-                /*String::class -> return "containsString(\"${(resContentsItem as String)
-                        .replace("\"", "\\\"")
-                        .replace("\n", "\\n")
-                        .replace("\r", "\\r")}\")"*/
-                //Note: checking a string can cause (has caused) problems due to unescaped quotation marks
-                // The above solution should be refined.
+                LinkedTreeMap::class -> return "NotCoveredYet"
```

Comment:
do not use "magic" strings, but rather put them in a field (ie, don't copy&paste the same string several times). especially if you have other function that the use the value "NotCoveredYet" in its if statements

Reply:
Done.